src/screens/Calendar.js

```diff
@@ -454,6 +546,106 @@ const CalendarPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="calendar.small.day.round">
+          <Description>
+            The amount to round the corner day when small.
```

Comment:
I'm not quite following the wording here. Should it be something like:
```suggestion
            The amount to round the corner of a day when small.
```

Reply:
^^ Yes we should use this wording